tools/UnityPaths/UnityPaths.cs

```diff
@@ -65,7 +83,7 @@ public static string TryGetUnityPath()
             }
 
             throw new Exception(
-                $"Could not find Unity in\n  {UnityHomeEnvVar}={UnityHomeValue}\n  {improbableUnityPath}\n  {hubPath}\n  {DefaultUnityPath}");
+                $"Could not find Unity in\n  [{improbableUnityPath}]\n  [{hubPath}]\n  [{DefaultUnityPath}]");
```

Comment:
```suggestion
                $"Could not find Unity in\n[{improbableUnityPath}]\n[{hubPath}]\n[{DefaultUnityPath}]");
```

nit: spaces are not really needed?

Reply:
It just indents the paths in the log files, rather unneeded.

Reply:
Will do